templates/default/docs/index.md

```diff
@@ -1,4 +1,5 @@
 ---
+theme: alt
```

Comment:
This change isn’t related to the issue, and note the two templates have deviated. Revert this part, or add it to both?

Reply:
good catch, sorry for this; it was unfortunate combination of a copy/paste from templates/empty/docs/index.md and an incomplete proof-read